content/en/blog/_posts/2021-03-30-metrics-stability-ga/2021-03-30-metrics-stability-hits-ga.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: blog
+title: 'Kubernetes 1.21: Metrics Stability hits GA'
+date: 2021-03-30
```

Comment:
```suggestion
date: 2021-04-23
```
The date needs to be changed here and in the filename, but other than that the post looks good to go!

Reply:
ah one sec, lemme do that